File: python/set08/problems/Problem1.py
# Create a function to calculate the distance between two points defined by their x, y coordinates
x1 ,y1 = input("Enter the coordinates of the first point: ").split(",")
x2 , y2 = input("Enter the coordinates of the second point: ").split(",")
distance = ((int(x2) - int(x1))**2 + (int(y2) - int(y1))**2)**0.5

print(distance)

File: javascript/set08/Problems/Problem1.js
// Create a function to calculate the distance between two points defined by their x, y coordinates
let x1 = prompt("Enter x1:");
let y1 = prompt("Enter y1:");
let x2 = prompt("Enter x2:");
let y2 = prompt("Enter y2:");

let distance = ((x2 - x1)**2 + (y2 - y1)**2)**0.5

console.log(distance)